compiler/quilt/tools/command.py

```diff
@@ -879,84 +887,125 @@ def install(package, hash=None, version=None, tag=None, force=False):
 
     pkgobj = store.install_package(owner, pkg, response_contents)
 
-    with requests.Session() as s3_session:
-        total = len(response_urls)
-        for idx, (download_hash, url) in enumerate(sorted(iteritems(response_urls))):
-            print("Downloading %s (%d/%d)..." % (download_hash, idx + 1, total))
+    obj_queue = sorted(iteritems(response_urls), reverse=True)
```

Comment:
Curious, why reverse?

Reply:
`obj_queue` is actually a stack (just cause I can use a normal list for that) - so I reverse it so it shows up sorted when popping items off the end.